PITCHME.md

```diff
@@ -1,25 +1,30 @@
-WRF for WIMEA-ICT
+WRF Portlet
 ===
 
 ![](assets/wrf-logo.png)
 
 ## How to use it
```

Comment:
Presentation preview is here: [https://gitpitch.com/sci-gaia/wimea-ict-wrf-portlet/feature](url)